frontend/src/views/WheresMyGene/components/Beta/index.tsx

```diff
@@ -0,0 +1,17 @@
+import { StyledCallout, SubmitIssue } from "./style";
+
+export default function Beta(): JSX.Element {
+  return (
+    <StyledCallout intent="info" dismissed={false}>
+      This feature is in beta. If you have any suggestions or feedback, please{" "}
+      <SubmitIssue
+        href="https://github.com/chanzuckerberg/single-cell-data-portal/issues/new/choose"
+        target="_blank"
+        rel="noopener"
+      >
+        submit an issue on our GitHub
+      </SubmitIssue>
```

Comment:
Non-blocker, but I don't think GitHub issues are the right avenue for suggestions or feedback

Reply:
ah good call out! What's the alternative you have in mind?

CC: @signechambers1 

Thank you!

Reply:
Discussions!

Reply:
Gh Discussions!